app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the environment variable `API_KEY` is set before running the application. If it's not set, the API request will fail. Consider adding a check to handle the case where `API_KEY` is `None`.